core/src/main/res/values/strings.xml

```diff
@@ -294,6 +294,7 @@
     <string name="move_to_top_label">Move to top</string>
     <string name="move_to_bottom_label">Move to bottom</string>
     <string name="sort">Sort</string>
+    <string name="sort_automatically">Keep sorted</string>
```

Comment:
I think the string should instead be called `keep_sorted`. That's more clear.

Reply:
Ok, I will update the code accordingly.